File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    #capitalize() -> 문자열 첫글자만 대문자로 변환
    
    s = s.split(' ')
    for i in range(len(s)):
        s[i] = s[i].capitalize()
    print(s)
    answer = ' '.join(s)
    #print(s.capitalize())
        
            
    return answer

File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(string s) {
    string answer = "";
    // 공백, 숫자 toupper, tolower에 영향 안 받음
    answer += toupper(s[0]); // 첫 글자 대문자 만들어줌
    for(int i = 1; i < s.length(); i++){
        if(s[i-1] == ' ') answer += toupper(s[i]); // 공백 뒤에 대문자 만들어줌
        else answer += tolower(s[i]);
    }
    return answer;
}